File: Python/AnggotaDPR.py
class AnggotaDPR:
    # Attribute Declaration
    __id = "" # Private ID
    __name = "" # Private Name
    __div_name = "" # Private Div
    __party_name = "" # Private Party
    
    # Constructor
    def __init__(self, id="n/a", name="n/a", div_name="n/a", party_name="n/a"):
        self.__name = name
        self.__id = id
        self.__div_name = div_name
        self.__party_name = party_name
    
    # Get Method for AnggotaDPR
    def getId(self):
        return self.__id
    def getName(self):
        return self.__name
    def getDiv(self):
        return self.__div_name
    def getParty(self):
        return self.__party_name
    # Set Method for AnggotaDPR
    def setId(self, id):
        self.__id = id 
    def setName(self, name):
        self.__name = name 
    def setDiv(self, div_name):
        self.__div_name = div_name
    def setParty(self, party_name):
        self.__party_name = party_name

File: Java/AnggotaDPR.java
package Java;

public class AnggotaDPR {
    // Private Attributes
    private String id; // ID for Members
    private String name; // Name for Members
    private String div_name; // Division for Members
    private String party_name; // Party name for members
    
    // Constructors
    public AnggotaDPR() {
        this.id = "n/a";
        this.name = "n/a";
        this.div_name = "n/a";
        this.party_name = "n/a";
    }
    public AnggotaDPR(String id, String name, String div_name, String party_name) {
        this.id = id;
        this.name = name;
        this.div_name = div_name;
        this.party_name = party_name;
    }
    
    // Get Methods
    public String getId() {
        return this.id;
    }
    public String getName() {
        return this.name;
    }
    public String getDiv() {
        return this.div_name;
    }
    public String getParty() {
        return this.party_name;
    }
    
    // Set Methods
    public void setId(String id){
        this.id = id;
    }
    public void setName(String name){
        this.name = name;
    }

    public void setDiv(String div_name){
        this.div_name = div_name;
    }
    public void setParty(String party_name){
        this.party_name = party_name;
    }
}
